modules/force_rtl.py

```diff
@@ -30,7 +29,7 @@ def force_rtl(drone: dronekit.Vehicle) -> bool:
     rtl_command = generate_command.return_to_launch()
 
     # Change drone mode
-    drone.mode = dronekit.VehicleMode("RTL")
+    drone.mode = drone.set_flight_mode("RTL")
```

Comment:
remove drone.mode, flightcontroller does not have mode as a member and we do not want to access the dronekit's internal mode this way.

take a look at how flightcontroller (more specifically its functions) is implemented at https://github.com/UWARG/common/blob/main/mavlink/modules/flight_controller.py

set_flight_mode returns a boolean